test/e2e/e2e_test.go

```diff
@@ -1719,27 +1731,65 @@ func waitForPodRunning(ctx context.Context, t *testing.T, clientSet clientset.In
 	}
 }
 
-func waitForPodsRunning(ctx context.Context, t *testing.T, clientSet clientset.Interface, labelMap map[string]string, desireRunningPodNum int, namespace string) {
-	if err := wait.PollUntilContextTimeout(ctx, 10*time.Second, 60*time.Second, true, func(ctx context.Context) (bool, error) {
+func waitForPodsRunning(ctx context.Context, t *testing.T, clientSet clientset.Interface, labelMap map[string]string, desireRunningPodNum int, namespace string) string {
```

Comment:
s/desireRunningPodNum/desiredRunningPodNum/

Reply:
If `waitForPodsRunning` returns a list of running pods instead of a single one then the caller can decide whether to take only the first pod or a different one (or more). Currently, returning a non-empty string is a corner case. With returning all the running pods the return value will be a non-corner case output. Less to worry about when refactoring the code in the future.